SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs

```diff
@@ -12,7 +15,15 @@ public class GameController : Controller
     {
         private ApplicationDbContext db = new ApplicationDbContext();
 
+        //Dictionary containing ESRB ratings
+        public static List<string> esrbList = new List<string>
+            {
+                {"EC"},{"E"},{"E10"},{"T"},{"M"},{"AO"}
```

Comment:
The way it is now we can only show the user the ESRB Code and not the more friendly ESRB labels. That's why I thought a dictionary might be more useful.